sdk/core/Azure.Core.Experimental/src/MutableJsonDocument.ChangeTracker.cs

```diff
@@ -55,39 +57,6 @@ internal bool DescendantChanged(string path, int highWaterMark)
                 return changed;
             }
 
-            internal bool TryGetChange(ReadOnlySpan<byte> path, out MutableJsonChange change)
```

Comment:
Removing this to tidy up; will reinstate it when it can be implemented [using a preferred approach](https://github.com/annelo-msft/azure-sdk-for-net/blob/core-json-utf8span/sdk/core/Azure.Core.Experimental/src/MutableJsonDocument.ChangeTracker.cs#L61).